torchvision/models/_api.py

```diff
@@ -115,7 +115,7 @@ def get_weight(name: str) -> WeightsEnum:
 W = TypeVar("W", bound=WeightsEnum)
 
 
-def get_model_weights(model: Union[Callable, str]) -> W:
+def get_model_weights(name: Union[Callable, str]) -> W:
```

Comment:
Curious what the reason is to change this? `name` is less accurate to describe this parameter IMO, since it can also be a callable / model builder. For example

```py
from torchvision.models import resnet50
weights = get_model_weights(name=resnet50)
```

looks unnatural.

Reply:
The `load()` method already a has a `model` parameter. Keeping it as `model` leads to `TypeError`:
```
TypeError: load() got multiple values for argument 'model'
```

Given that the specific parameter can be either string or callable, I think it's OK. If you have a better name, let me know.